# Generally we use pass in try...catch and also creating a block for future use

for i in range(10):
    pass

if i > 5:
    pass

print("some useful work")
